Sources/DistributedActorsTestKit/TestProbes.swift

```diff
@@ -47,6 +48,8 @@ public final class ActorTestProbe<Message: Codable>: @unchecked Sendable {
     internal let internalRef: _ActorRef<ProbeCommands>
     internal let exposedRef: _ActorRef<Message>
 
+    private lazy var _internal = _TestProbeInternal(actorSystem: self.system)
```

Comment:
lazy var is not thread safe, so that's another reason I was hoping we can initialize it during the test probe init, rather than like this

Reply:
I made it `lazy` because I noticed it gets created even when not needed and that messes up the traversal tests. Maybe one option is to add a flag to the initializer and make `_internal` optional?

Reply:
Ok I see, that's the opposite way around we need to be fixing though -- the traversal tests don't matter I'll check which ones :)

Reply:
I.e. this should be eager, especially since the current take is not entirely threadsafe and while it should nto matter too often in tests accessing the probe they must be threadsafe to access from anywhere :)

Reply:
ok, i will delete the traversal test and always initialize this in `init`.

Reply:
actually, the traversal test passes with `private let _internal: _TestProbeInternal`, so i will keep the test for now.

Reply:
ok :)